51-android.rules

```diff
@@ -959,12 +958,9 @@ LABEL="adb", ENV{adb_adb}="yes", GOTO="android_usb_rule_match"
 
 # ADB Debug and Audio Source
 LABEL="adbaud", ENV{adb_adb}="yes"
-LABEL="aud", ENV{adb_user}="yes", GOTO="android_usb_rule_match"
```

Comment:
I see you want to get rid of the LABEL="aud" since there is no tests (above) that use it.
The GOTO is used by both lines. Just copy the GOTO to line above, and then this line is okay being deleted.